scripts/integrity.js

```diff
@@ -30,51 +31,60 @@ function exists(file) {
 }
 
 // bootswatch
-var bootswatch = buildPath(config.bootswatch.bootstrap);
-for (var i = 0; i < config.bootswatch.themes.length; i++) {
-    var theme = config.bootswatch.themes[i];
-    var file  = bootswatch.replace('SWATCH_VERSION', config.bootswatch.version)
-                          .replace('SWATCH_NAME',    theme.name);
-
-    if (exists(file)) { // always regenerate
-        config.bootswatch.themes[i].sri = sri.digest(file);
+(function() {
+    var bootswatch = buildPath(config.bootswatch.bootstrap);
+    for (var i = 0; i < config.bootswatch.themes.length; i++) {
+        var theme = config.bootswatch.themes[i];
+        var file = bootswatch.replace('SWATCH_VERSION', config.bootswatch.version)
+                              .replace('SWATCH_NAME', theme.name);
+
+        if (exists(file)) { // always regenerate
+            config.bootswatch.themes[i].sri = sri.digest(file);
+        }
     }
-}
+})();
 
 // bootlint
-for (var i = 0; i < config.bootlint.length; i++) {
-    var bootlint = config.bootlint[i];
-    var file     = buildPath(bootlint.javascript);
+(function() {
```

Comment:
Why the closures? I ask, not to be contrary, but more for my own education. I never know when they're appropriate and when they're not.